scripts/install-forms-flow.sh

```diff
@@ -35,35 +35,75 @@ main() {
+        if [[ $is_multitenant =~ ^[Yy]$ ]]; then
+            # Commands for premium users with multitenancy enabled
+            helm upgrade --install forms-flow-ai forms-flow-ai --set Domain=$domain_name --set postgresql-ha.postgresql.podSecurityContext.enabled=true --set mongodb.podSecurityContext.enabled=true --set insight_api_key=$insight_api_key --set imageCredentials.registry=docker.io --set imageCredentials.username=$premium_username --set imageCredentials.password=$access_token --set forms-flow-idm.keycloak.EnableKeycloakClientAuth=true --set forms-flow-web.EnableMultitenant=true --set forms-flow-idm.realm=multitenant --set EnableChatBot=true -n $namespace --version $version_ff_ai
+            read -p "Include forms-flow-analytics? (y/n):" include_analytics
+            if [[ $include_analytics =~ ^[Yy]$ ]]; then
+                helm upgrade --install forms-flow-analytics forms-flow-analytics --set ingress.ingressClassName=$classname --set ingress.hosts[0].host=forms-flow-analytics-$namespace.$domain_name --set ingress.tls[0].secretName="forms-flow-analytics-$namespace.$domain_name-tls" --set ingress.tls[0].hosts[0]="forms-flow-analytics-$namespace.$domain_name" --set ingress.hosts[0].paths[0]="/" -n $namespace --kubeconfig/tmp/eks-dev/config --version $version_ff_bpm
+             fi
+            # Call external script to get Redash API Key
+            getRedashApiKey $classname
+            # Store Redash API key in a variable
+            REDASH_API_KEY=$?
+            echo "Redash API Key: $REDASH_API_KEY"
+            # Step 3: Re-run forms-flow-ai with the Redash API Key
+            helm upgrade --install forms-flow-ai forms-flow-ai --set Domain=$domain_name --set postgresql-ha.postgresql.podSecurityContext.enabled=true --set mongodb.podSecurityContext.enabled=true --set insight_api_key=$REDASH_API_KEY --set imageCredentials.registry=docker.io --set imageCredentials.username=$premium_username --set imageCredentials.password=$access_token --set forms-flow-idm.keycloak.EnableKeycloakClientAuth=true --set forms-flow-web.EnableMultitenant=true --set forms-flow-idm.realm=multitenant --set EnableChatBot=true -n $namespace --version $version_ff_ai
+            # Step 4: Re-run analytics if chosen
+            if [[ $include_analytics =~ ^[Yy]$ ]]; then
+                helm upgrade --install forms-flow-analytics forms-flow-analytics --set ingress.ingressClassName=$classname --set ingress.hosts[0].host=forms-flow-analytics-$namespace.$domain_name --set ingress.tls[0].secretName="forms-flow-analytics-$namespace.$domain_name-tls" --set ingress.tls[0].hosts[0]="forms-flow-analytics-$namespace.$domain_name" --set ingress.hosts[0].paths[0]="/" -n $namespace --kubeconfig/tmp/eks-dev/config --version $version_ff_bpm
```

Comment:
can you please set multiorg command as true

Reply:
is it another set command?